src/pages/workspaces/workspace/Data.js

```diff
@@ -58,17 +61,17 @@ class WorkspaceData extends Component {
       this.setState({ entityMetadata })
 
       if (this.state.selectedDataType) {
-        this.loadData()
+        this.loadData(true)
```

Comment:
Doing this here means that the page is reset to 1 if you refresh the browser page. In order to do this safely without extraneous calls, we might need to rearrange things a bit.